### ISSUE TITLE
screen reader is announcing  tree name

### ISSUE THREAD

**Issue by balajip8811** [microsoft/vscode#261295]
Type: <b>Bug</b>

when depth of tree is more than 8  screen reader nvda is  reading the tree name instead of file name in file explorer
same is happening in gitlense  commit , stash and compare view.
As i am a team lead facing lot of issue with it as our project has lot of depth.
so please fix the tree accessibility issue in reading  the nodes 
expected to read node name instead of tree name in vscode with screen reader at least in  accessibility mode is turned on.thanks

VS Code version: Code 1.103.0 (e3550cfac4b63ca4eafca7b601f0d2885817fd1f, 2025-08-06T21:40:10.271Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (8 x 1690)|
|GPU Status|2d_canvas: enabled<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.41GB (8.71GB free)|
|Process Argv|--crash-reporter-id 84cbae96-f29d-4378-ad85-f35db01fcda4|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (52)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-openapi|42C|4.38.0
Bookmarks|ale|13.5.0
copy-relative-path|ale|0.0.2
vscode-apollo|apo|2.6.2
turbo-console-log|Cha|3.4.2
csvtojson-converter|Chu|1.0.2
gitlens|eam|17.3.4
prettier-vscode|esb|11.0.0
codespaces|Git|1.17.3
copilot|Git|1.350.0
copilot-chat|Git|0.30.0
remotehub|Git|0.64.0
vscode-github-actions|git|0.27.2
vscode-pull-request-github|Git|0.88.0
vscode-graphql|Gra|0.13.2
vscode-graphql-syntax|Gra|1.3.8
git-graph|mhu|1.30.0
microsoft-testing|ms-|0.1.17
azure-dev|ms-|0.9.0
vscode-azure-github-copilot|ms-|1.0.76
vscode-azure-mcp-server|ms-|0.5.4
vscode-azureappservice|ms-|0.26.3
vscode-azurecontainerapps|ms-|0.9.0
vscode-azurefunctions|ms-|1.18.1
vscode-azureresourcegroups|ms-|0.11.1
vscode-azurestaticwebapps|ms-|0.13.1
vscode-azurestorage|ms-|0.16.5
vscode-azurevirtualmachines|ms-|0.6.9
vscode-containers|ms-|2.1.0
vscode-cosmosdb|ms-|0.27.1
vscode-docker|ms-|2.0.0
remote-containers|ms-|0.422.1
remote-ssh|ms-|0.120.0
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.99.0
azure-repos|ms-|0.40.0
remote-explorer|ms-|0.5.0
remote-repositories|ms-|0.42.0
remote-server|ms-|1.5.2
vscode-node-azure-pack|ms-|1.7.0
vscode-speech|ms-|0.16.0
windows-ai-studio|ms-|0.18.3
git-config-user-profiles|onl|2.1.0
vscode-jest|Ort|6.4.4
postman-for-vscode|Pos|1.15.0
vscode-thunder-client|ran|2.37.5
vscode-yaml|red|1.18.0
sonarlint-vscode|Son|4.28.0
rainbow-bracket|tal|1.0.2
vscode-ai-foundry|Tea|0.8.0
vscode-mermaid-editor|tom|0.19.1
json2yaml|tux|0.2.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vswsl492:30256859
binariesv615:30325510
nativeloc1:31344060
dwcopilot:31170013
6074i472:31201624
dwoutputs:31242946
copilot_t_ci:31333650
e5gg6876:31282496
pythoneinst12:31285622
c7cif404:31314491
996jf627:31283433
pythonrdcb7:31342333
usemplatestapi:31297334
0aa6g176:31307128
jjjhb125:31275178
aj953862:31281341
pylancequickfixf:31358881
9d2cg352:31346308
usemarketplace:31343026
nesew2to5:31336538
agentclaude:31366295
4f60g487:31327383
replacestringexc:31350595
nes-set-on:31351930
6abeh943:31336334
envsactivate1:31353494
yijiwantestdri0626-t:31336930
0927b901:31350571
f76d9909:31348711
45650338:31358607
0cj2b977:31352657

```

</details>

**Comment by meganrogge** [comment]
are you talking about when you're in the explorer and navigating folders/files?

**Comment by balajip8811** [comment]
@meganrogge  Yes,   it should announce file name but it is announcing tree name. also same for git lense tree like commit ,compare trees also.

**Comment by balajip8811** [comment]
I can not share the repo as it is private but recorded speech history output 
providers is in level 6 folder and it is expanded 
i just pressed down arrow  repeted so that it go through the files in  level 7
pasted the speech output below.
it is telling tree name (file explorer tree view) for every second file instead of file name 

providers  not selected  expanded  5 of 6  level 6
Files Explorer  tree view
level 7  address.service.spec.ts  not selected  2 of 7
Files Explorer  tree view
giftcard.service.spec.ts  not selected  4 of 7  level 7
Files Explorer  tree view
users.service.spec.ts  not selected  6 of 7  level 7
Files Explorer  tree view
level 6  users.module.ts  not selected  6 of 6
level 5  validation  not selected  collapsed  9 of 12

**Comment by balajip8811** [comment]
@meganrogge  sorry i mistakenly closed and reopened the issue required info also pasted in the above comment. if u need any other info please mention.

**Comment by brunoprietog** [comment]
I'm facing this one too 😢 it's really hard to navigate through those files

**Comment by meganrogge** [comment]
If this is a confirmed regression, it would help to know which change caused it. We maintain a node.js based CLI tool  [`vscode-bisect`](https://github.com/microsoft/vscode-bisect) that automatically downloads and runs previous VS Code insiders and asks for each build if the issue reproduces. It takes up to 8 steps to find exactly the build that caused the regression.

**Steps:**
* install [node.js](https://nodejs.org/en/download) (if not yet done)
* `npx --yes @vscode/vscode-bisect@latest`
* follow the instructions until you found the offending build (its fine to leave the commit empty)
* report back the commit range in this issue

Thanks!

**Comment by brunoprietog** [comment]
It's been a long time, I think even years. My workaround so far has been to have another window open with Windows Explorer, and then open the file with the command palette. I thought it was just me, but we know that's not the case

**Comment by balajip8811** [comment]
@meganrogge  I am totally confused here
it is totaly working in insider build
Version: 1.102.0-insider
Commit: bdc1dd9a7c36b7dab7a9eeec2f05a19f7abd0409
Date: 2025-06-20T09:31:30.580Z
Electron: 35.5.1
ElectronBuildId: 11727614
Chromium: 134.0.6998.205
Node.js: 22.15.1
V8: 13.4.114.21-electron.0
OS: Windows_NT x64 10.0.22631



But not working in the below version . i disabled all extentions and restarted vscode
also i can see all extentions in  disabled in extentions section also
Version: 1.103.0 (user setup)
Commit: e3550cfac4b63ca4eafca7b601f0d2885817fd1f
Date: 2025-08-06T21:40:10.271Z
Electron: 37.2.3
ElectronBuildId: 12035395
Chromium: 138.0.7204.100
Node.js: 22.17.0
V8: 13.8.500258-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by balajip8811** [comment]
@meganrogge  Could the reason be because of  layout of panels editors and primary and secondary bars??

**Comment by lramos15** [comment]
@balajip8811 That insiders you posted is an old one. See the date 

> Date: 2025-06-20T09:31:30.580Z

Can ou try the newest insiders. As this could indicate this broke sometime between Date: 2025-06-20T09:31:30.580Z and Date: 2025-08-06T21:40:10.271Z

**Comment by balajip8811** [comment]
How to do it? i am running with --latest flag

**Comment by balajip8811** [comment]
it seems all builds are good i checked but failing even after disabling all extentions in noramal vscode 
[build] All builds are good! Try running with --releasedOnly to support older builds.

**Comment by balajip8811** [comment]
@lramos15  @meganrogge  what can be done now?

**Comment by meganrogge** [comment]
What you said is confusing. Is it fixed or not? Which versions work vs not?

**Comment by balajip8811** [comment]
@lramos15  @meganrogge  I am also confused
it is working in vscode insider build perfectly
even i installed github copilot and gitlense also 
but not working in normal vscode
1.103.0 also 1.103.1 latest release
both versions of normal vscode i tested not working in normal vscode it seems
I even changed profile to python and node js still not working in normal vscode
I tested removing settings.json and keyboard.json and disabling all extentions still not working properly.
i changed themes also but no luck
I am completely out of my options to find root cause.

**Comment by meganrogge** [comment]
That means it was likely fixed! Thanks for the info. Closing.
